### PR TITLE
Correction of the plug_connected_boiler value.

### PR DESCRIPTION
**Pull request by clinique**
and added temperature trend and pressure trend readings.
Signed-off-by: Gaël L'hopital <glhopital@gmail.com>

### CODE REVIEW

**Comment by cbornet** [comment]
I read https://github.com/openhab/openhab2-addons/issues/2379 . This inconsistency of Netatmo API is really annoying, especially when using typed mappings ...
Can you also make your changes to swagger_access_token.json ?

**Comment by clinique** [comment]
Yes, it's really annoying. On the other these fields are undocumented, so we must accept it like it is....

File: swagger.json [hunk 1]
```diff
@@ -1324,7 +1324,7 @@
                     "format": "int32"
                 },
                 "plug_connected_boiler": {
-                    "type": "boolean"
+                    "type": "int32"
```
Comment (IOOOTAlan) [inline swagger.json]: Shouldn't this be
```json
  "type": "integer",
  "format": "int32"
```
?

Comment (IOOOTAlan) [inline swagger.json]: I think that the `NAPlug.plug_connected_boiler` field needs the change from boolean to int, too.
I've got conversion issues with gson until I changed it too...

Comment (cbornet) [inline swagger.json]: @IOOOTAlan it should. I surely need to integrate a CI test with swagger validator

Comment (IOOOTAlan) [inline swagger.json]: @clinique do you mind a PR for this small fix?

Comment (cbornet) [inline swagger.json]: @clinique @IOOOTAlan CI is in place. You can see the error at https://travis-ci.org/cbornet/netatmo-swagger-decl/builds/262997598#L590

**Comment by clinique** [comment]
@cbornet : CI now works with last modification. Thanks to @IOOOTAlan

**Comment by cbornet** [comment]
Thanks @clinique @IOOOTAlan !

**Pull request merged**